lib/globalize/active_record/instance_methods.rb

```diff
@@ -12,14 +12,15 @@ def attributes
       end
 
       def attributes=(new_attributes, *options)
-        return unless new_attributes.is_a?(Hash)
-        assign_attributes(new_attributes, *options)
+        super unless new_attributes.respond_to?(:stringify_keys) && new_attributes.present?
+        attributes = new_attributes.stringify_keys
+        with_given_locale(attributes) { super(attributes.except("locale"), *options) }
```

Comment:
Note that we could just call `assign_attributes` here, but then if some other gem decided they wanted to override just `attributes=` we would be bypassing that. I figure this level of code duplication is justified to avoid that.